src/ddscxx/include/org/eclipse/cyclonedds/topic/datatopic.hpp

```diff
@@ -23,16 +23,40 @@
 #include "dds/ddsi/q_radmin.h"
 #include "dds/ddsi/ddsi_serdata.h"
 #include "dds/ddsi/ddsi_sertopic.h"
+#include "basic_cdr_ser.hpp"
```

Comment:
When the number of supported stream types will be expanded, each will need to have their header included separately, maybe we should bundle these in its own definition header?